### PR TITLE
Change marker to use ovsdb

### PR DESCRIPTION
**Pull request by SchSeba**
This PR change the bridge-marker to use the ovsdb and not the
ovs-vsctl binary.

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/ovs-cni/80)

### CODE REVIEW

File: cluster/examples/openshift-ovs-cni.yml [hunk 1]
```diff
@@ -26,31 +26,6 @@ spec:
       - key: node-role.kubernetes.io/master
         operator: Exists
         effect: NoSchedule
-      initContainers:
-      - name: ovs-vsctl
-        command:
-        - /bin/bash
-        - -c
-        - |
-          cp /usr/bin/ovs-vsctl /host/usr/bin/_ovs-vsctl
```
Comment (phoracek) [inline cluster/examples/openshift-ovs-cni.yml]: Bless you @SchSeba, this scripts was oooooof.

File: cluster/examples/openshift-ovs-cni.yml [hunk 2]
```diff
@@ -26,31 +26,6 @@ spec:
       - key: node-role.kubernetes.io/master
         operator: Exists
         effect: NoSchedule
-      initContainers:
```
Comment (phoracek) [inline cluster/examples/openshift-ovs-cni.yml]: I think we don't need separate manifest for OpenShift and for Kubernetes. Could you please check whether they are the same now?

File: pkg/marker/marker.go [hunk 3]
```diff
@@ -17,20 +17,18 @@ package marker
...
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
-)
 
-var (
-	clientset kubernetes.Interface
+	"github.com/kubevirt/ovs-cni/pkg/ovsdb"
 )
 
+var ()
```
Comment (phoracek) [inline pkg/marker/marker.go]: Why empty vars?

File: pkg/ovsdb/ovsdb.go [hunk 4]
```diff
@@ -23,15 +23,34 @@ import (
...
 // Create a new OVS driver with Unix socket
+func NewOvsDriver(ovsSocket string) (*OvsDriver, error) {
+	ovsDriver := new(OvsDriver)
+
+	ovsDB, err := libovsdb.ConnectWithUnixSocket(ovsSocket)
+	if err != nil {
+		return nil, fmt.Errorf("failed to connect to ovsdb error: %v", err)
+	}
+
+	// Setup state
```
Comment (phoracek) [inline pkg/ovsdb/ovsdb.go]: I don't understand what do you want to say with this comment

Comment (SchSeba) [reply pkg/ovsdb/ovsdb.go]: it's a leftover sorry

**Comment by phoracek** [comment]
The OpenShift CI has passed tests, but stuck on cleanup of the job. Crossing my fingers and merging.

**Pull request merged**